### PR TITLE
fix: make controller param final and deploy to VMs

### PR DESCRIPTION
**Pull request by JosephKelly**
Also deploy image for this commit if it is tagged as latest.  Abort the build if it is no longer the latest.
squash! chore: on master, wait up to 15 minutes for an image to be built

TISNEW-5376: Enable TIS Admins to see submitted Form Rs

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  def healthcheckEndpoint="/${serviceName}/actuator/health"
+  def version
+  def isDeployable = false
+  def branchName = null
+  def changeId = null
+
+  stage('Checkout') {
+    version = sh(returnStdout: true, script: "git log -n 1 --pretty=format:'%H'").trim()
+    //clone the repo
+    if(env.BRANCH_NAME.startsWith('PR-')) {
```
[the remark this answers is not in the document]
Comment (JosephKelly) [reply Jenkinsfile:13]: TBH it was grabbed from another build.  Given what the script is doing I am inclined to deregister the webhook for PRs (just use pushes)

[the remark this answers is not in the document]
Comment (JosephKelly) [reply Jenkinsfile:13]: ... to master

[the remark this answers is not in the document]
Comment (JosephKelly) [reply Jenkinsfile:13]: Just realised this isn't going to work.  Jenkins doesn't seem to be populating `env.BRANCH_NAME`

[the remark this answers is not in the document]
Comment (Judge40) [reply Jenkinsfile:13]: Fair enough, hopefully this is short term anyway.

**Comment by JosephKelly** [comment]
I think there are better solutions now.

**Pull request closed without merging**